src/session.ts

```diff
@@ -21,7 +20,9 @@ interface StoreConfig {
 }
 
 interface BotStats {
+  _id: string;
   keys: string[];
+  [K: `keys.${number}`]: string;
```

Comment:
What's this?

Reply:
See https://github.com/grammyjs/free-session-backend/blob/fb82b903f361e93fb2153b2e72bbf3a487881b1e/src/session.ts#L98